sass/components/_buttons.scss

```diff
@@ -202,21 +206,24 @@
     &.halfway-fab {
       bottom: -56px * 0.5;
     }
+
     width: 56px;
     height: 56px;
     padding: 0;
-    // font-size: 
+    // font-size:
     // i {
     //   // line-height: 56px;
     // }
   }
 
   &.btn-small {
     --btn-small-height: calc(0.75 * var(--btn-height));
+
     &.halfway-fab {
       bottom: calc(var(--btn-small-height) * -0.5);
     }
-    width: var(--btn-small-height)e;
+
+    width: var(--btn-small-height) e;
```

Comment:
the `e` in the back looks like a typo too.

Reply:
can you remove it please, then i will merge it. Thank you.

Reply:
Typo fixed ✓